docs/source/models.rst

```diff
@@ -533,9 +533,6 @@ MLflow can deploy models locally as local REST API endpoints or to directly scor
 MLflow can package models as self-contained Docker images with the REST API endpoint. The image can
 be used to safely deploy the model to various environments such as Kubernetes.
 
-You deploy MLflow model locally or generate a Docker image using the CLI interface to the
```

Comment:
@shivp950 This also should not be removed.

Reply:
we haven't updated the cli to use the latest version of model deploy for azureml, if we keep the build image and deploy cli commands, it's using the deprecated aml image.deploy method

Reply:
@shivp950 The CLI we're referring to in this part of the documentation has nothing to do with AzureML! It should be left as-is!

Reply:
yes I see now, my bad, I'll fix it